src/app/app.module.ts

```diff
@@ -1,16 +1,46 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http'
 
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { catReducer } from './State/reducer';
+import { CatEffects } from './State/effects';
+
+import { AppRoutingModule } from './routes/app-routing.module';
```

Comment:
app-routing.module is related to module files and generally it's just 1 file, so we don't need to have routes folder